packages/mdx/test/index.test.js

```diff
@@ -86,20 +86,39 @@ COPY start.sh /home/start.sh
 
 it('Should support comments', async () => {
   const result = await mdx(`
-A paragraph
 <!-- a Markdown comment -->
+A paragraph
+
+Some text <!-- an inline comment -->
+
 \`\`\`md
-<!-- a code block Markdown comment -->
+<!-- a code block string -->
 \`\`\`
+
 <div>
   {/* a nested JSX comment */}
-  <!-- a nested Markdown comment -->
+  <!-- div content -->
```

Comment:
This would technically cause a Babel error, but I included it in the test to ensure that it doesn't turn into a comment.